apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -215,7 +240,23 @@ export default connect(
           if (typeof output === 'object') {
```

Comment:
What should happen if output is not an object? Should an error be thrown?

Reply:
Great point! Based on the structure we decided to move forward with, all inputs/outputs coming from the debug console and the code workspace in app lab should now all be in the form of an object of {input: input} and etc. So I believe this logic of checking if it's an object should actually be removed. Instead, if output.toJS() fails, which means it's not an object, then it should be rescued then just pass the output into the react-inspector.

I've already made that change to the PR for game lab but would love to hear feedback on it as well. I wrote that logic like below:

`try {
              output = output.toJS();
            } catch (error) {
              return <Inspector key={i} data={output} />;
            }`

Reply:
Note: Looks like this is blocked on a different PR: https://github.com/code-dot-org/code-dot-org/pull/28856/files 

What would be an example of when this wouldn't be an object?

Reply:
Discussed with Jessie in person and concluded there aren't examples of when it's not an object but it is a valid logOutput. Also, for any comments on this section of the PR in the future please refer to this PR: https://github.com/code-dot-org/code-dot-org/pull/28856.